tools/autograd/templates/Functions.cpp

```diff
@@ -1961,6 +1961,38 @@ Tensor log1p_backward(const Tensor& grad, const Tensor& self) {
   return grad / (self + 1);
 }
 
+Tensor constant_pad_nd_backward(const Tensor& grad, const Tensor& self, IntList pad) {
+    auto l_inp = self.dim();
+    auto l_pad = pad.size() / 2;
+    auto l_diff = l_inp - l_pad;
+
+    auto grad_input = Variable(at::zeros(self.sizes(), grad.options()));
```

Comment:
nit: no need to use `Variable` wrapper here.

Reply:
👍